src/app/services/dashboard.js

```diff
@@ -137,7 +137,7 @@ function (angular, $, kbn, _, config, moment, Modernizr) {
                     self.dash_load(dashboard);
                     // No? Ok, grab default.json, its all we have now
                 } else {
-                    self.file_load('coinform');
+                    self.file_load('dev-fakeNewsNet.json');
```

Comment:
This changes the default production dashboard!!

Reply:
Please change back to `coinform` as that is the default dashboard in production. `dev-fakeNewsNet` is OK for development, but not suitable for production, so can't accept this for a pull-request to the production branch.

Reply:
Sorry, I changed it to develop the new functionalities and I forgot to restore the original dashboard.